<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ArenaInfoDialog</class>
    <widget class="QDialog" name="ArenaInfoDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>202</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="label">
                            <property name="text">
                                <string>Top</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>Next</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <widget class="QLabel" name="label_3">
                            <property name="text">
                                <string>Next free</string>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="0">
                        <widget class="QLabel" name="label_4">
                            <property name="text">
                                <string>System Memory</string>
                            </property>
                        </widget>
                    </item>
                    <item row="4" column="0">
                        <widget class="QLabel" name="label_5">
                            <property name="text">
                                <string>Max Memory</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" name="lineEditTop">
                            <property name="readOnly">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLineEdit" name="lineEditNext">
                            <property name="readOnly">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QLineEdit" name="lineEditNextfree">
                            <property name="readOnly">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="1">
                        <widget class="QLineEdit" name="lineEditSysMem">
                            <property name="readOnly">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item row="4" column="1">
                        <widget class="QLineEdit" name="lineEditMaxMem">
                            <property name="readOnly">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                    <item row="5" column="0">
                        <widget class="QLabel" name="label_6">
                            <property name="text">
                                <string>Last Remainder</string>
                            </property>
                        </widget>
                    </item>
                    <item row="5" column="1">
                        <widget class="QLineEdit" name="lineEditLastRem">
                            <property name="readOnly">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
